sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIClient.java

```diff
@@ -687,4 +715,499 @@ Response<BinaryData> getChatCompletionsWithAzureExtensionsWithResponse(
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return english language transcribed text and associated metadata from provided spoken audio data.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public AudioTranscription getAudioTranslationAsResponseObject(
+            String deploymentOrModelName, AudioTranslationOptions audioTranslationOptions) {
+        // Generated convenience method for getAudioTranslationAsResponseObjectWithResponse
+        RequestOptions requestOptions = new RequestOptions();
+        return getAudioTranslationAsResponseObjectWithResponse(
+                        deploymentOrModelName, BinaryData.fromObject(audioTranslationOptions), requestOptions)
+                .getValue()
+                .toObject(AudioTranscription.class);
+    }
+
+    /**
+     * Gets English language transcribed text and associated metadata from provided spoken audio data.
+     *
+     * @param deploymentOrModelName Specifies either the model deployment name (when using Azure OpenAI) or model name
+     *     (when using non-Azure OpenAI) to use for this request.
+     * @param audioTranslationOptions The configuration information for an audio translation request.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return english language transcribed text and associated metadata from provided spoken audio data.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public String getAudioTranslationAsPlainText(
+            String deploymentOrModelName, AudioTranslationOptions audioTranslationOptions) {
+        // Generated convenience method for getAudioTranslationAsPlainTextWithResponse
+        RequestOptions requestOptions = new RequestOptions();
+        return getAudioTranslationAsPlainTextWithResponse(
+                        deploymentOrModelName, BinaryData.fromObject(audioTranslationOptions), requestOptions)
+                .getValue()
+                .toObject(String.class);
+    }
```

Comment:
Should we customize them to make them package private?

I will make next typespec-java automatically make "multipart/form-data" be access=internal and convenientApi=false

Reply:
In java unfortunately we could not find a suitable way of using `@@internal/@@include` that would allow to have the models we need for the request as well. 

We need the models to be public but the operations to be private.

Reply:
You should be able to use `@@access(Azure.OpenAI.AudioTranscriptionOptions, Access.public, "java");`, after now root package.json is on 0.34.0 https://github.com/Azure/azure-rest-api-specs/blob/main/package.json#L6

Even without do this in typespec client.tsp, one can still make these API package private by removing the `@Generated` and delete `public`.